src/commands/module/add.ts

```diff
@@ -72,11 +72,10 @@ export default defineCommand({
         if (!config.modules) {
           config.modules = []
         }
-        for (let i = 0; i < config.modules.length; i++) {
-          if (config.modules[i] === r.pkgName) {
-            consola.info(`\`${r.pkgName}\` is already in the \`modules\``)
-            return
-          }
+       
+        if (config.modules.includes(r.pkgName)) {
```

Comment:
FYI, `config` is not a real JS array but an instance of MagicAST proxy. Calling `.includes`, runs [this method](https://github.com/unjs/magicast/blob/main/src/proxy/array.ts#L52) which almost has same performance.

But this is a nice refactor anyway since this feature was not initially available in magicast 👍🏼

Reply:
Thank you! Very kind. 💚🔥

 I was just checking the code to see how to contribute to allow multiple modules addition, when I saw that part and said "hey, I can make my first non-typo contribution! This is easy to fix!". GPTChat helped me with the benchmarks code 😂. Going to play with magicast, I'm using pathe in my projects.
 
 As always, great job you all have been doing with Nuxt/unjs.